website/server/index.js

```diff
@@ -22,9 +22,10 @@ const logger = require('./libs/logger').default;
 const IS_PROD = nconf.get('IS_PROD');
 const IS_DEV = nconf.get('IS_DEV');
 const CORES = Number(nconf.get('WEB_CONCURRENCY')) || 0;
+const NO_CLUSTER = Number(nconf.get('NO_CLUSTER')) || 0;
```

Comment:
if this is a boolean - I don't think we should use `Number()` here

but otherwise I don't know if we'd need this new config, I think if you just set `WEB_CONCURRENCY` to 0 in your config.json this should just work out fine

Reply:
nice, removed NO_CLUSTER env